src/frontends/onnx/frontend/src/input_model.cpp

```diff
@@ -193,6 +193,27 @@ void InputModel::set_element_type(const ov::frontend::Place::Ptr& place, const n
     m_editor->set_input_types(m);
 }
 
+ov::element::Type InputModel::get_element_type(const ov::frontend::Place::Ptr& place) const {
+    std::string tensor_name;  // name of the model input which should be reshaped
+    const auto input_edge = std::dynamic_pointer_cast<PlaceInputEdge>(place);
+    const auto output_edge = std::dynamic_pointer_cast<PlaceOutputEdge>(place);
+    if (input_edge) {
+        const auto tensor_names = input_edge->get_source_tensor()->get_names();
+        OPENVINO_ASSERT(!tensor_names.empty(),
+                        "Cannot retrieve source tensor name for this InputEdge and thus partial shape.");
+        tensor_name = tensor_names[0];
+    } else if (output_edge) {
```

Comment:
From the parameter's freezing perspective the output edge handling is not required. This code can still stay here for potential future use.

Reply:
Let leave it here then